.circleci/config.yml

```diff
@@ -688,4 +689,4 @@ workflows:
             branches:
               only: master
             tags:
-              only: /^v.*/
+              only: /^v[0-9]+\.[0-9]+\.[0-9]+$/
```

Comment:
I don't think we should document prereleases in the changelog, as it would drown out the changes for a stable release.
I think it would be better to handle these manually!